includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -82,14 +118,7 @@ public function register() {
 		$synchronize_publication->register();
 
 		if ( Feature_Flags::enabled( 'rrmModuleV2' ) && $this->is_connected() ) {
-			$post_meta       = new Post_Meta();
-			$publication_id  = $this->get_settings()->get()['publicationID'];
-			$post_product_id = new Post_Product_ID( $post_meta, $publication_id );
-			$post_product_id->register();
-
-			$term_meta       = new Term_Meta();
-			$term_product_id = new Term_Product_ID( $term_meta, $publication_id );
-			$term_product_id->register();
```

Comment:
Term metadata is no longer required. See https://github.com/google/site-kit-wp/issues/9961#issuecomment-2641401056.

Reply:
That's a good point. I've removed `Term_Product_ID` but left out `Term_Meta` as it is more of a utility class and something we may need in the future, similar to how `Post_Meta` persisted since TwG days. Thanks, Tom!

Reply:
Makes sense, thanks @nfmohit!